app/coffee/UpdateManager.coffee

```diff
@@ -39,15 +39,15 @@ module.exports = UpdateManager =
 			return callback(error) if error?
 			if updates.length == 0
 				return callback()
-			UpdateManager.applyUpdates project_id, doc_id, updates, callback
+			async.mapSeries updates,
```

Comment:
if return values aren't needed, **async.eachSeries** instead of **.map** avoids building up an array of them.